.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
I understand that `package-lock.json` cannot be published to the npm registry (it's ignored at publishing), and will therefore also be ignored when it is npm-installed by packages that depend on `livepush` (like `balena-cli`). This can lead to dependency-related errors being masked when `livepush` is tested in isolation. Reference: https://docs.npmjs.com/files/package-lock.json